scripts/asciidoc2html.py

```diff
@@ -30,19 +31,26 @@
 import tempfile
 import argparse
 import io
+import pathlib
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir))
 
 from scripts import utils
 
+DOC_DIR = pathlib.Path("qutebrowser/html/doc")
+
 
 class AsciiDoc:
 
     """Abstraction of an asciidoc subprocess."""
 
     FILES = ['faq', 'changelog', 'contributing', 'quickstart', 'userscripts']
 
-    def __init__(self, asciidoc, website):
+    def __init__(self, asciidoc, website) -> None:
+        """
+        asciidoc: Optional[List[str]]
+        website: Optional[List[str]])
+        """
```

Comment:
Please use the annotation syntax (`asciidoc: Optional[List[str]]` etc.) instead - that is supported in all Python 3 versions, only annotating variables isn't.